ding/rl_utils/tests/test_td.py

```diff
@@ -216,6 +256,38 @@ def test_dist_1step_compatible():
     assert pytest.approx(nstep_loss.item(), onestep_loss.item())
 
 
+@pytest.mark.unittest
+def test_dist_1step_multi_agent_td():
```

Comment:
validate the following two cases are equivalent:
- call `dist_1step_td_error` with `(B, A, *)` shape one time
- call `dist_1step_td_error` with `(B, *)` shape `agent` times